aperturedb/Connector.py

```diff
@@ -94,14 +98,20 @@ class Connector(object):
 
     def __init__(self, host="localhost", port=55555,
                  user="", password="", token="",
-                 use_ssl=True, shared_data=None, authenticate=True,
+                 use_ssl=True,
+                 shared_data=None,
+                 authenticate=True,
                  use_keepalive=True,
-                 retry_connect_interval_seconds=1,
-                 retry_connect_max_attempts=3,
+                 retry_interval_seconds=1,
+                 retry_max_attempts=3,
```

Comment:
We discussed adding a parameter so that the Connector is blocking, disabled by default, for use cases where we want to use the Connector to verify connection is possible (following current behaviour). Up to you if you want to make it a parameter before merging, or on a separate PR.

Reply:
opened this issue for putting this . https://github.com/aperture-data/aperturedb-python/issues/369.

Will merge this PR for now, and see any other implications.